docs/program/thermal_conductivity.md

```diff
@@ -67,765 +67,302 @@ Optional switches:
 \end{equation}
 $$
 
+This approximation consists in neglecting the collective phonon contribution to the thermal conductivity tensor and can also be obtain by decoupling the four-point correlation in product of two-point correlation.
+
+It is important to note that while starting from differents considerations, this formulation and the Boltzmann equation [^peierls1929]<sup>,</sup>[^peierls1955quantum]<sup>,</sup>[^Broido2007]<sup>,</sup>[^Broido2005] are strictly equivalent [^Fiorentino2023].
 
 
-### Off-diagonal / coherence's contribution
+### The off-diagonal contribution
 
-The Boltzmann equation only takes into account the relaxation of phonons from perturbed state to the equilibrium.
-However, for systems with complex unit cell, a contribution stemming from coherence tunneling between different phonons ($\lambda \neq \lambda'$) can become important.
-This off-diagonal term can be introduced both by a formulation based on the Hardy current [^Isaeva2019] or a Wigner current [^Simoncelli2019], with very similar results[^Caldarelli2022].
-In TDEP, we use the formulation based on the Hardy heat current [^Isaeva2019]
+The off diagonal heat tensor is written
 
 $$
 \begin{equation}
-J_\alpha = \frac{1}{V} \sum_{\lambda\lambda'} \frac{\omega_\lambda + \omega_{\lambda'}}{2} v_{\lambda\lambda'}^\alpha \hat{a}_\lambda^\dagger \hat{a}_{\lambda'} \delta_{q_{\lambda} q_{\lambda'}}
+J_{\alpha}^{\mathrm{nd}} = \sum'_{\lambda \lambda'} \hbar \omega_{\lambda'} v_{\lambda\lambda'}^{\alpha} A_\lambda B_{\lambda'}
 \end{equation}
 $$
 
-where $v_{\lambda\lambda'}^\alpha$ are generalized group velocity tensor [^Dangic2021]
+where $\sum'$ indicates that $\lambda = \lambda'$ is excluded from the sum.
+Injecting this contribution into the Green-Kubo formula also ends up in something proportional to a four-point correlation function
 
 $$
 \begin{equation}
-v_{\lambda\lambda'}^\alpha = \frac{i}{2 \sqrt{\omega_\lambda \omega_{\lambda'}}} \sum_{ij \beta\gamma} \epsilon_\lambda^{i\beta} \sum_{\mathbf{R}} \big( R^\alpha \frac{\Phi_{ij}^{\beta\gamma}(\mathbf{R})}{\sqrt{m_i m_j}} \big) \epsilon_{\lambda'}^{j\gamma}
+\kappa_{\alpha\beta}^\mathrm{nd} \propto \int_0^\infty dt \int_0^\beta d\lambda \langle A_{\lambda}(i\hbar\lambda) B_{\lambda'}(i\hbar\lambda) A_{\lambda''}(t) B_{\lambda'''}(t) \rangle
 \end{equation}
 $$
 
-with $\mathbf{R}$ the distance vector between unit cells and $m_i$ the mass of atom $i$ in the unit cell.
-It should be recognized that for $\lambda = \lambda'$, we recover the heat current used previously to derive the thermal conductivity with the Boltzman equation.
-
-The derivation of this off-diagonal contribution starts from the Green-Kubo formula for the thermal conductivity tensor
+For this contribution, we will directly neglect the collective part and decouple the four-point correlation in product of two-point correlations
```

Comment:
The discussion in appendix B is a bit subtle. The standard decoupling can be used without any problem for spectral properties, but since we are working with Kubo correlations, this correspond to some semi-classical approximation. To get to the results we have for the mode-coupling paper, in practice we do a quantum -> classical -> quantum transformation.
This allowed us to obtain the quantum correlation function because any quantum correlation function gives back the unique classical limit.

The new decoupling scheme avoid this double transformation, and is important to get the quantum heat capacity for the thermal conductivity. Without it, we would have the classical heat capacity, so $k_B$.